src/game.js

```diff
@@ -0,0 +1,60 @@
+function Game() {
+  this.rolls = [];
+  this.score = 0;
+  this.frame = 1;
+  this.frameRollCount = 1;
+}
+
+Game.prototype.roll = function(roll) {
```

Comment:
There's a lot of mentions of "frame", both in the code and in the variable names.  Does this tell give us a hint about whether the code is adhering to the SRP?